Makefile

```diff
@@ -112,7 +128,9 @@ e2e-test: e2e-build
 		-e CI=$(CI) \
 		-v $(PWD)/e2e/playwright-report:/e2e/playwright-report \
 		-v $(PWD)/e2e/blob-report:/e2e/blob-report \
-		playwright-e2e
+		playwright-e2e | grep -v "To open last HTML report run" | grep -v "npx playwright show-report"
+	@echo "📄 HTML report generated!"
+	@echo "🔍 View it with: make e2e-show-report"
```

Comment:
remove the default playwright command it shows and insert our make command

Reply:
What about not removing the the playwright command but just echo'ing the make command ?

Will move just the package updates over to `template-infra`

Reply:
@lorenyu see above comment if it makes sense to do

Reply:
> What about not removing the the playwright command but just echo'ing the make command ?

Sure that sounds good